nats/context_command.go

```diff
@@ -35,7 +35,11 @@ type ctxCommand struct {
 func configureCtxCommand(app *kingpin.Application) {
 	c := ctxCommand{}
 
-	context := app.Command("context", "Manage nats configuration contexts").Alias("ctx")
+	context := app.Command("context", `Manage nats configuration contexts
+
+Context names can not contain your OS path separator or ".." but are otherwise
+at your discretion.  There is one selected context which is recorded as
+default, and the NATS_CONTEXT environment variable can override that.`).Alias("ctx")
```

Comment:
I think probably updating main.go with

https://github.com/nats-io/jetstream/blob/2734c559e8090f59be4e0995f14b266935228c39/nats/main.go#L52

adding `Envar("NATS_CONTEXT").PlaceHolder("NATS_CONTEXT")` will achieve most of whats in this PR?

It will if the variable is set have as if --context was passed and so override whatever was stored.

Raging headache today so I might be missing something obvious but seems so to me :)

Reply:
That's probably a much cleaner implementation, I didn't even check to see if this CLI lib supported env var defaults for flags.  I'll leave the PR open for now, to act as a reminder/nudge.